data/transactions/logic/eval_test.go

```diff
@@ -3716,18 +3840,14 @@ func TestAllowedOpcodesV2(t *testing.T) {
 					strings.Contains(err.Error(), "illegal opcode") ||
 						strings.Contains(err.Error(), "pc did not advance"),
 				)
-				_, err = CheckStateful(program, ep)
-				require.Error(t, err, source)
-				require.True(t,
-					strings.Contains(err.Error(), "illegal opcode") ||
-						strings.Contains(err.Error(), "pc did not advance"),
-				)
 				_, err = Eval(program, ep)
 				require.Error(t, err, source)
 				require.Contains(t, err.Error(), "illegal opcode")
-				_, _, err = EvalStateful(program, ep)
-				require.Error(t, err, source)
-				require.Contains(t, err.Error(), "illegal opcode")
+
+				// It is not necessary to test CheckStateful and EvalStateful
```

Comment:
I think it is better to leave `EvalStateful/CheckStateful` tests and call it with `ep. MinTealVersion = 1`. In this case another code path is triggered and actual opcodes version check happen.

Reply:
Done, actually at some point I made a change to this PR that made changing this test not necessary.